tests/python/test_fields_builder.py

```diff
@@ -3,174 +3,205 @@
 
 import taichi as ti
 
+'''
+Test fields with shape.
+'''
 
-@ti.test(arch=[ti.cpu, ti.cuda, ti.vulkan, ti.metal])
+
+@ti.test(arch=ti.get_host_arch_list())
```

Comment:
The original archs (`ti.cpu, ti.cuda, ti.vulkan, ti.metal`) for testing should not be removed.

Reply:
I refered other tests and changed from `ti.cpu, ti.cuda, ti.vulkan, ti.metal` to `arch=ti.get_host_arch_list()`.

Reply:
```python
def get_host_arch_list():
    return [_ti_core.host_arch()]
```

However I not found the python impl. of the `host_arch()`, it seems a C++ method?

Reply:
I changed back from `arch=ti.get_host_arch_list()` to `ti.cpu, ti.cuda, ti.vulkan, ti.metal`